### PR TITLE
Add fuzzers for linux loader

### PR DESCRIPTION
**Pull request by likebreath**
Signed-off-by: Bo Chen <chen.bo@intel.com>

### CODE REVIEW

File: fuzz/Cargo.toml [hunk 1]
```diff
@@ -75,6 +76,12 @@ path = "fuzz_targets/iommu.rs"
 test = false
 doc = false
 
+[[bin]]
+name = "load_kernel"
```
Comment (rbradford) [inline fuzz/Cargo.toml:80]: I think this (and the file) should be called `linux_loader` as that is what is fuzzing.

**Review by sboeuf — APPROVED** [review]
Looking good! Please address @rbradford's comment and we'll be good to merge your PR!

File: fuzz/Cargo.toml (hunk 1 above)
Comment (likebreath) [reply fuzz/Cargo.toml]: Okay. I intended to make it differentiate from the other fuzzer from the `linux_loader`  crate (initially named as `loader_kernel_cmdline`). Now renamed both. PTAL.

**Comment by likebreath** [comment]
@sboeuf I also pushed another fuzzer `linux_loader_cmdline` to the PR. PTAL.

**Review by sboeuf — APPROVED** [review]

**Pull request merged**